Source/CUNYAIModule.cpp

```diff
@@ -286,6 +290,10 @@ void CUNYAIModule::onFrame()
         land_inventory = mineral_inventory + geyser_inventory; // for first initialization.
         current_map_inventory.updateBaseLoc(land_inventory);
         current_map_inventory.getExpoPositions(); // prime this once on game start.
+		
+		// Don't let overlords scout against terran. Currently the only modification to AI based off race.
+		Race enemyRace = Broodwar->enemy()->getRace();
```

Comment:
We should probably store this in the Player_Model. It is going to get crowded in there, but it is better in there than out here, I suppose.